### PR TITLE
Clean up Orbitalism

### PR DESCRIPTION
**Pull request by deutschbitte**

### CODE REVIEW

File: orbitalism/index.html [hunk 1]
```diff
@@ -83,16 +83,16 @@ <h3>Contributors</h3>
...
+		<p><strong>Réka Kovács</strong>, <a href="http://www.linkedin.com/in/kovacsr">Co-Founder</a> at <a href="http://www.nanosatisfi.com/">Nanosatisfi</a>. Advised on the current state of the space industry, microgravity experience, and space tourism. Co-creation of ideas for a space tourist user experience.</p>
+		<p><strong>Anders Højmose</strong>, Interaction Designer at TraceAgency. Project advisor.</p>
 		<p><strong>Matt Cottam</strong>, CEO at <a href="http://tellart.com/">Tellart</a>. Advised on service design and user-centric design for astronauts.</p>
 		<p><strong>Stefan Eder</strong>, Motion and 3D Designer at <a href="http://flugfeld53.at/">Flugfeld53</a>. 3D visualization.</p>
-		<p><strong><a href="http://www.dchris.net/">Christian Lendl</a></strong>, Multimedia Engineer and Tourism Specialist. Interview about the tourism industry and co-creation of ideas for a space tourist offering.</p>	
+		<p><strong>Christian Lendl</strong>, <a href="http://www.dchris.net/">Multimedia Engineer and Tourism Specialist</a>. Interview about the tourism industry and co-creation of ideas for a space tourist offering.</p>
```
Comment (deutschbitte) [inline orbitalism/index.html:92]: I had some names linked but don't love how that looks so I linked their titles. But let me know if you have a preference.

File: orbitalism/index.html [hunk 2]
```diff
@@ -83,16 +83,16 @@ <h3>Contributors</h3>
...
-		<p><strong>Franz Viehböck</strong>, <a href="http://de.wikipedia.org/wiki/Franz_Viehb%C3%B6ck">Former Astronaut</a>. Sojus TM-12 Mission to Mir (1991). Interview and user testing and co-creation of ideas for a space tourist user experience.</p>			
+		<p><strong>Franz Viehböck</strong>, <a href="http://de.wikipedia.org/wiki/Franz_Viehb%C3%B6ck">Former Astronaut,</a> Soyuz TM-12 Mission to Mir (1991). Interview and user testing and co-creation of ideas for a space tourist user experience.</p>			
 		<p><strong>Dr. Werner Balogh</strong>, Programme Officer at <a href="http://www.oosa.unvienna.org/oosa/index.html">UN Office for Outer Space Affairs</a>. Advised on international space law and the current state of the space industry.</p>
-		<p><strong><a href="http://www.linkedin.com/in/kovacsr">Réka Kovács</a></strong>, Co-Founder at <a href="http://www.nanosatisfi.com/">Nanosatisfi</a>. Advised on the current state of the space industry, microgravity experience, and space tourism. Co-creation of ideas for a space tourist user experience.</p>
-		<p><strong>Anders Højmose</strong>, Interaction Designer at <a href="http://www.traceagency.com">TraceAgency</a>. Project advisor.</p>
```
Comment (deutschbitte) [inline orbitalism/index.html:89]: traceagency.com doesn't exist so I removed the link. Not sure if there's another that should be added.

File: orbitalism/index.html [hunk 3]
```diff
@@ -83,16 +83,16 @@ <h3>Contributors</h3>
...
+		<p><strong>Réka Kovács</strong>, <a href="http://www.linkedin.com/in/kovacsr">Co-Founder</a> at <a href="http://www.nanosatisfi.com/">Nanosatisfi</a>. Advised on the current state of the space industry, microgravity experience, and space tourism. Co-creation of ideas for a space tourist user experience.</p>
```
Comment (deutschbitte) [inline orbitalism/index.html:88]: http://www.nanosatisfi.com/ now redirects to https://spire.com/. Not sure if you still want to link or whatever.

File: orbitalism/index.html (hunk 2 above)
Comment (smeidu) [reply orbitalism/index.html:89]: let's link to his new personal website instead. https://www.hojmo.se/

File: orbitalism/index.html (hunk 3 above)
Comment (smeidu) [reply orbitalism/index.html:88]: let's keep that one as is. she was co-founder at the time of this project.

**Pull request merged**